.azure/agents/functional.json

```diff
@@ -15,7 +15,7 @@
         {
             "name": "windows-powershell-invokecommand",
             "parameters": {
-               "Script": "verifier.exe /standard /driver xdp.sys fnmp.sys fnlwf.sys ndis.sys ebpfcore.sys; exit 0; <# Enable verifier on XDP-related drivers. Mask the verifier.exe non-zero exit code. #>"
+               "Script": "verifier.exe /standard /driver xdp.sys xdpfunctionaltestdrv.sys fnmp.sys fnlwf.sys ndis.sys ebpfcore.sys; exit 0; <# Enable verifier on XDP-related drivers. Mask the verifier.exe non-zero exit code. #>"
```

Comment:
This needs to be merged in a separate PR along with matching changes to prepare-machine.ps1.